client/src/components/paginationTable/PaginationTable.js

```diff
@@ -50,7 +50,7 @@ const PaginationTable = (props) => {
 
```

Comment:
Thanks to the filters and the respectable amount of series, I found out we have some bug when the following scenario happens:
Assume you are on page 3, and your filter causes the totalElements to be 11 (= 2 pages). It'll try to fetch `pageNumber=3` and get `{content: [], totalElements: 11}` and display nothing but with 2 pages. So we have to do something like this:

```
const loadRequest = useCallback(() => {
        loadRequestFn(currentPage)
            .then(response => {
                const totalCount = response.totalElements;
                const lastPage = Math.ceil(totalCount / pageSize);

                if (totalCount > 0 && currentPage > lastPage) {
                    setCurrentPage(lastPage); // or maybe setCurrentPage(1)? what seems more legit?
                } else {
                    setTotalCount(totalCount);
                    setCurrentData(response.content);
                }
            });
    }, [loadRequestFn, currentPage]);
```

If we encounter the scenario above, it'll firstly fetch `pageNumber=3` and will realize based on `totalElements` that it exceeded the last page. Then it'll fetch `pageNumber=lastPage` properly.